### PR TITLE
Added checkr application

### PR DESCRIPTION
**Pull request by skc0201**

### CODE REVIEW

File: src/app/app.component.css
Comment (abdulkudduszemoso) [inline src/app/app.component.css:1]: Remove this file if not needed

File: src/app/atoms/chip/chip.component.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+@Component({
+  selector: 'app-chip',
+  templateUrl: './chip.component.html',
+  styleUrls: ['./chip.component.css']
+})
+export class ChipComponent implements OnInit {
+  @Input() text: string="Hello";
+  @Input() type: string="green";
+
```
Comment (abdulkudduszemoso) [inline src/app/atoms/chip/chip.component.ts:11]: Is this component been used anywhere?

File: src/app/organisms/court-searches/court-searches.component.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+})
+export class CourtSearchesComponent implements OnInit {
+  displayedColumns: string[] = [ 'search', 'status', 'date' ];
+  readonly COURTSERACHES_HEADER =COURTSERACHES_HEADER;
+  readonly COURT_SEARCHES = COURT_SEARCHES;
+  dataSource = new MatTableDataSource<CourtSearch>(DATA);;
+
+  constructor() { }
+
+  ngOnInit(): void {
```
Comment (abdulkudduszemoso) [inline src/app/organisms/court-searches/court-searches.component.ts]: Do we need this?

File: src/app/organisms/court-searches/court-searches.component.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  templateUrl: './court-searches.component.html',
+  styleUrls: ['./court-searches.component.css']
+})
+export class CourtSearchesComponent implements OnInit {
+  displayedColumns: string[] = [ 'search', 'status', 'date' ];
+  readonly COURTSERACHES_HEADER =COURTSERACHES_HEADER;
+  readonly COURT_SEARCHES = COURT_SEARCHES;
+  dataSource = new MatTableDataSource<CourtSearch>(DATA);;
+
+  constructor() { }
```
Comment (abdulkudduszemoso) [inline src/app/organisms/court-searches/court-searches.component.ts]: Do we need this? Check for other places also

**Review by abdulkudduszemoso — APPROVED** [review]

File: src/app/atoms/chip/chip.component.ts (hunk 1 above)
Comment (skc0201) [reply src/app/atoms/chip/chip.component.ts:11]: yes , in Candidate detail page

**Pull request merged**